journals/apps/journals/models.py

```diff
@@ -126,7 +126,8 @@ def get_discovery_data(self):
             'access_length': self.journal.access_length.days,
             'card_image_url': self.journal_about_page.card_image.file if self.journal_about_page.card_image else '',
             'short_description': self.journal_about_page.short_description,
-            'full_description': self.journal_about_page.long_description
+            'full_description': self.journal_about_page.long_description,
+            'status': 'active' if self.journal_about_page.live else 'inactive',
```

Comment:
just a question:
might this field be better as "active" with value of True|False?
Then we wouldn't have to compare string values when checking the status but instead just testing boolean
I'm fine either way, just a suggestion

Reply:
I was basing it off of the way Courses and Programs has status. It also allows for us to add more options down the road if need be.